apps/webapp/src/components/shared/toast-content.tsx

```diff
@@ -14,6 +14,20 @@ export const loadingTxToast: ToastFnProps = {
   duration: Infinity, // Building could potentially take a long time. Expected to be dismissed manually.
 };
 
+export const broadcastingTxToast = (): ToastFnProps => ({
+  main: 'Broadcasting transaction',
+  variant: 'default',
+  //subText: shorten(txHash, 8),
+  action: (
```

Comment:
Don't think we should have an action button for this stage. It too quickly disappears and it goes to a table that isn't responsive when a new transaction comes in.

Given the short length of time, we should just have another spinner like: https://loading.io/spinner/radio/-alien-connection-signal-radio-rss-wifi-information-transmit-sync-transfer with the new text.